fala/playwright/setup.py

```diff
@@ -16,3 +18,59 @@ def tearDownClass(cls):
         super().tearDownClass()
         cls.browser.close()
         cls.playwright.stop()
+
+    def visit_results_page(self, postcode, checkbox_labels=None):
+        if checkbox_labels is None:
+            checkbox_labels = []
+        page = self.browser.new_page()
+        page.goto(f"{self.live_server_url}")
+        page.get_by_label("Postcode").fill(postcode)
+        for label in checkbox_labels:
+            page.get_by_label(label).check()
+        page.get_by_role("button", name="Search").click()
+        if page.locator("#changeSearchButton"):
+            return ResultsPage(page)
+        else:
+            return OtherRegionPage(page)
+
+    def visit_results_page_with_full_search(self, postcode, organisation, checkbox_labels):
+        page = self.browser.new_page()
+        page.goto(f"{self.live_server_url}")
+        page.get_by_label("Postcode").fill(postcode)
+        page.get_by_label("Organisation name").fill(organisation)
+        for label in checkbox_labels:
+            page.get_by_label(label).check()
+        page.get_by_role("button", name="Search").click()
+        if page.locator("#changeSearchButton"):
+            return ResultsPage(page)
+        else:
+            return OtherRegionPage(page)
+
+    def visit_other_region_page(self, postcode):
+        page = self.browser.new_page()
+        page.goto(f"{self.live_server_url}")
+        page.get_by_label("Postcode").fill(postcode)
+        page.get_by_role("button", name="Search").click()
+        expect(page.locator("p")).to_have_text("This search only covers England and Wales")
+        return OtherRegionPage(page)
+
+    def visit_search_page(self):
+        page = self.browser.new_page()
+        page.goto(f"{self.live_server_url}")
+        return SearchPage(page)
+
+    def invalid_postcode_search(self):
+        page = self.browser.new_page()
+        page.goto(f"{self.live_server_url}")
+        page.get_by_label("Postcode").fill("ZZZ1")
```

Comment:
i would be tempted to have the bulk of these methods in the actual test rather than the setup - isn't this the thing we are testing? We're kind of hiding it by having it here. Same for the invalid org search one.